docs/en/tutorials/runner.md

```diff
@@ -1,3 +1,523 @@
+
+    # OptimizerWrapper, new concept in MMEngine for richer optimization options
+    # Default value works fine for most cases. You may check our documentations
+    # for more details, e.g. 'AmpOptimWrapper' for enabling mixed precision
+    # training.
+    optim_wrapper=dict(
+        optimizer=dict(
+            type=Adam,
+            lr=0.001)),
+    # ParamScheduler to adjust learning rates or momentums during training
+    param_scheduler=dict(
+        type='MultiStepLR',
+        by_epoch=True,
+        milestones=[4, 8],
+        gamma=0.1),
+
+    # validation data
+    val_dataloader=DataLoader(
+        dataset=MyDataset(
+            is_train=False,
+            size=1000),
+        shuffle=False,
+        collate_fn=default_collate,
+        batch_size=1000,
+        pin_memory=True,
+        num_workers=2),
+    # validation configurations, usually leave it an empty dict
+    val_cfg=dict(),
+    # evaluation metrics and evaluator
+    val_evaluator=dict(type=Accuracy),
+
+    # following are advanced configurations, try to default when not in need
+    # hooks are advanced usage, try to default when not in need
+    default_hooks=dict(
+        # the most commonly used hook for modifying checkpoint saving interval
+        checkpoint=dict(type='CheckpointHook', interval=1)),
+
+    # `luancher` and `env_cfg` responsible for distributed environment
+    launcher='none',
+    env_cfg=dict(
+        cudnn_benchmark=False,   # whether enable cudnn_benchmark
+        backend='nccl',   # distributed communication backend
+        mp_cfg=dict(mp_start_method='fork')),  # multiprocessing configs
+    log_level='INFO',
+
+    # load model weights from given path. None for no loading.
+    load_from=None
+    # resume training from the given path
+    resume=False
+)
+
+# start training your model
+runner.train()
+```
+
+</details>
+
+### Explanations on example codes
+
+Really a long piece of code, isn't it! However, if you read through the above example, you may have already understood the training process in general even without knowing any implementation details, thanks to the compactness and readability of runner codes (probably). This is what MMEngine expects: a structured, modular and standardized training process that allows for more reliable reproductions and clearer comparisons.
+
+The above example may lead you to the following confusion:
+
+<details>
+<summary>There are too many arguments!</summary>
+
+Don't worry. As we mentioned before, **use runner as a memo**. The runner covers all aspects just to ensure you won't miss something important. You don't actually need to configure everything. The simple example in [15 minutes](../get_started/15_minutes.md) still works fine, and it can be even more simplified by removing `val_evaluator`, `val_dataloader` and `val_cfg` without breaking down. All configurable arguments are driven by your demands. Those not in your focus usually works fine by default.
```

Comment:
```suggestion
Don't worry. As we mentioned before, **use runner as a memo**. The runner covers all aspects just to ensure you won't miss something important. You don't actually need to configure everything. The simple example in [15 minutes](../get_started/15_minutes.md) still works fine, and it can be even more simplified by removing `val_evaluator`, `val_dataloader`, and `val_cfg` without breaking down. All configurable arguments are driven by your demands. Those not in your focus usually work fine by default.
```